__tests__/app.test.js

```diff
@@ -37,67 +37,104 @@ describe('Endpoint Tests', () => {
+                })
+            });
 
-    });
-    describe('/api/articles/:article_id', () => {
-        test('GET 200   | Return 200 and correct object when passed id', () => {
-            return request(app).get('/api/articles/3').expect(200).then(({body}) => {
-                expect(body.article.article_id).toBe(3)
-                expect(body.article).toHaveProperty("author")
-                expect(body.article).toHaveProperty("title")
-                expect(body.article).toHaveProperty("body")
-                expect(body.article).toHaveProperty("topic")
-                expect(body.article).toHaveProperty("created_at")
-                expect(body.article).toHaveProperty("votes")
-                expect(body.article).toHaveProperty("article_img_url")
-            })
         });
-        test("GET 404   | Return 400 and sends an appropriate error message when given a valid but non-existent id", () => {
-            return request(app).get('/api/articles/10000').expect(404).then(({body}) => {
-                expect(body.msg).toBe('article does not exist')
-            })
+        describe('/api/articles/:article_id', () => {
+            test('GET 200   | Return 200 and correct object when passed id', () => {
+                return request(app).get('/api/articles/3').expect(200).then(({body}) => {
+                    expect(body.article.article_id).toBe(3)
+                    expect(body.article).toHaveProperty("author")
+                    expect(body.article).toHaveProperty("title")
+                    expect(body.article).toHaveProperty("body")
+                    expect(body.article).toHaveProperty("topic")
+                    expect(body.article).toHaveProperty("created_at")
+                    expect(body.article).toHaveProperty("votes")
+                    expect(body.article).toHaveProperty("article_img_url")
+                })
+            });
+            test("GET 404   | Return 404 and sends an appropriate error message when given a valid but non-existent id", () => {
+                return request(app).get('/api/articles/10000').expect(404).then(({body}) => {
+                    expect(body.msg).toBe('article does not exist')
+                })
+            });
+            test('GET 400   | Return 400 and sends an appropriate error message when given an invalid id', () => {
+                return request(app).get('/api/articles/three').expect(400).then(({body}) => {
+                    expect(body.msg).toBe('invalid id')
+                })
+            });
         });
-        test('GET 400   | Return 400 and sends an appropriate error message when given an invalid id', () => {
-            return request(app).get('/api/articles/three').expect(400).then(({body}) => {
-                expect(body.msg).toBe('invalid id')
-            })
+        describe('/api/articles/:article_id/comments', () => {
+            test('GET 200   | Return 200 and array with correct comment objects', () => {
```

Comment:
We would still expect a test with a 200 if the article exists, when there are no comments